templates/zerver/api/roles-and-permissions.md

```diff
@@ -0,0 +1,85 @@
+# Roles and permissions
+
+Zulip offers [several levels of permissions based on user
+organization-level roles](/help/roles-and-permissions).
+
+Here are some important details to note when working with these
+roles and permissions in Zulip's API:
+
+## A user's organization-level role
```

Comment:
This version works fine, though. I think in "user organization-level roles", I get garden-pathed into wanting "user organization" to be a phrase.

Reply:
Actually, I think shortening to 'A user's role' would work as I work through these revisions.

There's one other part of the API that uses the word 'role' as a property / in regards to streams (is the user a subscriber or an admin of the stream). But if we're clear in the documentation for those instances and have this page to link to for clarification, I think it will be fine.